Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
If this is an application-wide truth (there is no link because people are not longer authorized to edit that), then a Voter is the right solution. If it isn't… well this would be a bit strange, wouldn't it? In short : by thinking you can always find a new reason why this could be a wanted feature. What I'm saying is : don't support it until it is really needed, and when it is needed, ask yourself if this is the best way to support it (maybe there should be a separate abstraction that determines if a link can be seen or not, like a Voter that would not be a **security** voter).